src/tests/test_util.h

```diff
@@ -23,15 +22,15 @@
   }
 #define ASSERT(expr)                                                                       \
   if (!(expr)) {                                                                           \
-    fprintf(stderr, "%s:%d: Assertion '%s' Failed\n", __FILE__, __LINE__, __STRING(expr)); \
+    fprintf(stderr, "%s:%d Assertion '%s' Failed\n", __FILE__, __LINE__, __STRING(expr));  \
     return -1;                                                                             \
   }
 
 #define ASSERT_STRING_EQ(s1, s2) ASSERT(!strcmp(s1, s2));
 
-#define ASSERT_EQUAL_INT(x, y, ...)                                 \
+#define ASSERT_EQUAL(x, y, ...)                                 \
   if (x != y) {                                                     \
-    fprintf(stderr, "%s:%d: ", __FILE__, __LINE__);                 \
+    fprintf(stderr, "%s:%d: Assertion Failed " __VA_ARGS__ ": ", __FILE__, __LINE__);                 \
```

Comment:
@mheiber cool, so can you please just refactor the testing macro and verify that it works properly, and that all tests are passing, and I'll merge?

re runes - it is possible since "ss" is represented by two 32 bit wide characters, and we'll take just the first one. But again, let's disregard for now.